keymanager/src/api/requests.rs

```diff
@@ -0,0 +1,115 @@
+use std::default::Default;
```

Comment:
This should be imported as part of the default prelude and so an explicit import should not be needed?